xwiki-platform-distribution/xwiki-platform-distribution-flavor/xwiki-platform-distribution-flavor-test/xwiki-platform-distribution-flavor-test-security/pom.xml

```diff
@@ -44,6 +44,12 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-test-rest</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
```

Comment:
Don't you also need to add a dependency to `xwiki-platform-extension-security-ui` since it's not in the flavor anymore ?

Reply:
Good catch, thanks!

Reply:
`xwiki-platform-extension-security-ui` added as a `provided` dependency